addons/account_peppol/wizard/peppol_registration_views.xml

```diff
@@ -0,0 +1,107 @@
+                    <field name="peppol_eas"/>
+                    <field name="peppol_endpoint"/>
+                    <field name="account_peppol_phone_number"/>
+                    <field name="account_peppol_contact_email"/>
+
+                    <!-- edi mode selection -->
+                    <field name="account_peppol_edi_mode"
+                           widget="account_peppol_radio_field"
+                           class="o_field_radio"
+                           readonly_items="account_peppol_mode_constraint != 'prod' and ['prod'] or []"
+                           hidden_items="account_peppol_mode_constraint != 'test' and ['test'] or []"
+                           options="{'horizontal': true }"/>
+                    <field name="account_peppol_mode_constraint" invisible="1"/>
+                    <div colspan="2" class="mt-3 mb-3" invisible="account_peppol_edi_mode != 'prod'">
+                        By clicking the button below I accept that Odoo may process my e-invoices.
+                    </div>
+                    <div colspan="2" class="mt-3 mb-3" invisible="account_peppol_edi_mode != 'test'">
+                        Test mode allows sending e-invoices through the test Peppol network.
+                        By clicking the button below I accept that Odoo may process my e-invoices.
+                    </div>
+                    <div colspan="2" class="mt-3 mb-3" invisible="account_peppol_edi_mode != 'demo'">
+                        In demo mode sending invoices is simulated.
+                        There will be no communication with the Peppol network.
+                    </div>
+                </group>
+
+                <!-- Step two. The participant needs to fill in the SMS code they received -->
+                <group name="step_verification" invisible="account_peppol_proxy_state != 'in_verification'">
+                    <field name="account_peppol_phone_number"/>
+                    <field name="account_peppol_contact_email"/>
+                    <div colspan="2" class="text-muted mt-3">
+                        We sent a verification code to
+                        <field name="account_peppol_phone_number"
+                               class="oe_inline"
+                               nolabel="1"
+                               readonly="account_peppol_proxy_state == 'in_verification'"/>.
+                    </div>
+                    <div colspan="2" class="mt-1 ps-3 w-50">
+                        <field name="account_peppol_verification_code" widget="verification_code"/>
+                    </div>
+                </group>
+
+                <!-- Final sender step. The participant is now a sender. Deregister button is visible in the footer -->
+                <group name="step_sender" invisible="account_peppol_proxy_state != 'sender'">
+                    <p colspan="2" class="mt-3">
+                        You can now send e-invoices via Peppol.
```

Comment:
When this field is the only text displayed on the view, it looks weird... huge padding, no button...

Reply:
After that step, you need to reopen the send & print... it's quite a bad UIX imo